src/server/rest/v1/router/api/ChargingStationRouter.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
```

Comment:
I really prefer to have eslint rules disablement at the line level ...
Maybe there's a tunable to avoid it on arrow function.